source/fuzz/fuzzer_pass_add_function_calls.cpp

```diff
@@ -241,10 +241,7 @@ std::vector<uint32_t> FuzzerPassAddFunctionCalls::ChooseFunctionCallArguments(
               true));
         }
       } else {
-        // TODO(https://github.com/KhronosGroup/SPIRV-Tools/issues/3177): We use
-        //  constant zero for the parameter, but could consider adding a fact
-        //  to allow further passes to obfuscate it.
-        result.push_back(FindOrCreateZeroConstant(arg_type_id));
+        result.push_back(FindOrCreateZeroConstant(arg_type_id, true));
```

Comment:
Please make this an irrelevant one.

I.e., for now: we pass something existing where possible, and pass an irrelevant zero otherwise, and we have a new issue to discuss whether we should always prefer to pass an irrelevant id.

Reply:
Fixed in 8d9e990.